crates/bevy_audio/Cargo.toml

```diff
@@ -25,7 +25,7 @@ bevy_utils = { path = "../bevy_utils", version = "0.14.0-dev" }
 rodio = { version = "0.17", default-features = false }
 
 [target.'cfg(target_os = "android")'.dependencies]
-oboe = { version = "0.5", optional = true }
```

Comment:
this shouldn't be updated, it's blocked on cpal/rodio releases

Reply:
Ok, we'll wait until then to update oboe